server/src/repos/RecipeRepository.ts

```diff
@@ -0,0 +1,148 @@
+import { Db, Collection, InsertOneResult, ObjectId } from "mongodb";
+import {
+  RecipePayload,
+  Recipe,
+  RecipePatchPayload,
+  RecipeCreateRequestPayload,
+} from "../interfaces/Recipe";
+
+export interface RecipeRepositoryInterface {
```

Comment:
you can name it RecipeRepository, without "Interface" and its current mongo implementation you can name as "RecipeMongoRepository"